### PR TITLE
Styling and polish

### PR DESCRIPTION
**Pull request by aomarks**
Live at https://aomarks-test-docs.appspot.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,12 +60,11 @@ The following custom properties and mixins are available for styling:
 
 Custom property | Description | Default
 ----------------|-------------|----------
-`--iron-doc-accent-color1` | Color for emphasis (e.g. hyperlinks). | `black`
-`--iron-doc-accent-color2` | Color for bright emphasis (e.g. hyperlink hover). | `black`
-`--iron-doc-font-code` | Mixin applied to code snippets. | `{}`
+`--iron-doc-accent-color1` | Color for emphasis (e.g. hyperlink hover). | ``
```
Comment (samuelli) [inline README.md]: color instead of color1?

File: iron-doc-viewer.html [hunk 2]
```diff
@@ -240,33 +240,26 @@
...
-            }
-            this._currentDescriptor = descriptor;
-            return;
+          var descriptorType, name, demoName;
+          if (this.baseHref && path.indexOf(this.baseHref) >= 0) {
+            path = path.substring(this.baseHref.length);
+          }
+          var parts = path.split('/');
+          if (parts.length > 1) {
+            [, descriptorType, name] = parts;
```
Comment (samuelli) [inline iron-doc-viewer.html]: es6, keep it es5 for now

**Review by samuelli — CHANGES_REQUESTED** [review]

File: iron-doc-viewer.html (hunk 2 above)
Comment (aomarks) [reply iron-doc-viewer.html]: Done.

**Review by samuelli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aomarks) [reply README.md]: Done.

**Pull request merged**